.gitignore

```diff
@@ -5,3 +5,4 @@
 *.csv
 *.txt
 *.png
+_targets
```

Comment:
You could add the `*/out/*` here so that you don't accidentally commit anything from those out directories.